eng/pipelines/libraries/helix-scenarios.yml

```diff
@@ -0,0 +1,34 @@
+parameters:
```

Comment:
I don't think we should introduce a new template just for this. 

Maybe make the project a parameter?

```yml
parameters:
   helixProject: `'$(Build.SourcesDirectory)/src/libraries/sendtohelix.proj`
  ...

steps:
  - script: ${{ parameters.msbuildScript }}
                 ${{ parameters.helixProject }}
                 ...
```

Reply:
I could do this, but if we decide to just augment the existing `sendtohelix.proj` with the new scenarios functionality, it won't be necessary.